packages/grafana-ui/src/components/Forms/InlineField.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC } from 'react';
+import { cx, css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
+import { getInlineLabelStyles, InlineLabel } from './InlineLabel';
+import { PopoverContent } from '../Tooltip/Tooltip';
+import { FieldProps } from './Field';
+
+export interface Props extends Omit<FieldProps, 'css'> {
+  /** Content for the label's tooltip */
+  tooltip?: PopoverContent;
+  /** Custom width for the label */
+  labelWidth?: number | 'auto';
+  /** Make the field's child to fill the width of the row. Equivalent to setting `flex-grow:1` on the field */
+  grow?: boolean;
+  /** A toggle to apply query keyword styling to the label */
+  isKeyword?: boolean;
+  /** Fill the remaining width of the row with the label's background */
+  fill?: boolean;
```

Comment:
what's the difference between grow and fill? Feels like we only need one of them?

Reply:
`grow` fills the rest of the row with a form element (input, etc):
![Screenshot 2020-09-24 at 14 07 50](https://user-images.githubusercontent.com/8878045/94137669-6f477380-fe6f-11ea-8dc0-67bee04385b0.png)
`fill` fills the rest of the row with dark background:
![Screenshot 2020-09-24 at 14 07 57](https://user-images.githubusercontent.com/8878045/94137699-7f5f5300-fe6f-11ea-9b71-5067499efa47.png)
But I also heard that we might be getting rid of the fill background, so I'm waiting for @jessover9000 feedback on this PR.